### ISSUE TITLE
actually_calculate_loss = False?

### ISSUE THREAD

**Issue by idealboy** [miraclewkf/multilabel-MXNet#8]
When I set actually_calculate_loss = True, I come accross the loss=nan

Now I train 1000 classes (4110000 train samples with 200 epoch )with this code, and I found the inference score is most lower than 0.5, Do you meet this chance?

Thank you!
